app/assets/stylesheets/lottery-results-pdf-generator.css

```diff
@@ -0,0 +1,344 @@
+	background-color: white;
+	table-layout: fixed;
+	width: 100%;
+}
+
+#lottery-results-pdf-column, #lottery-results-pdf-th {
+	font-family: Verdana, sans-serif;
+	text-align: left;
+	vertical-align: top;
+	/* this actually functions like a min-width */
+	width: 11ch;
+}
+
+#lottery-results-pdf-th-h4 {
+	font-size: 14px;
+  	text-transform: none;
+	font-family: Verdana, Sans-serif;
+	height: 2em;
+	font-weight: 700;
+  	letter-spacing: 0;	
+}
+
+#lottery-results-pdf-row {
+	font-family: Verdana, sans-serif!important;
+}
+
+#lottery-results-pdf-row > div {
+	font-family: Verdana, sans-serif;
+	padding-right: 1.25rem;
+	flex: 1;
+}
+
+#lottery-results-pdf-row > :nth-of-type(1) {
+	color: black;
+}
+
+#lottery-results-pdf-row > :nth-of-type(2) {
+	color: rebeccapurple;
+}
+
+#lottery-results-pdf-row > :nth-of-type(3) {
+	color: darkgreen;
+}
+
+#lottery-results-pdf-row > :nth-of-type(4) {
+	color: darkred;
+}
+
+#lottery-results-pdf-row > :nth-of-type(5) {
+	color: darkblue;
+}
+
+#lottery-results-pdf-row > :nth-of-type(6) {
+	color: darkorange;
+}
+
+#lottery-results-pdf-row > :nth-of-type(7) {
+	color: darkolivegreen;
+}
+
+#lottery-results-pdf-row > :nth-of-type(8) {
+	color: darkorchid;
+}
+
+#lottery-results-pdf-row > :nth-of-type(9) {
+	color: darkgoldenrod;
+}
+
+#lottery-results-pdf-th-h4, #lottery-results-pdf-column h5 {
+	margin: .5rem 0 .5rem 2rem;
+	flex-wrap: wrap;
+	display: flex;
+	font-family: Verdana, sans-serif;
+}
+
+#lottery-results-pdf-th-h4 {
+	height: 2em;
+	align-content: flex-end;
+}
+
+#lottery-results-pdf-column h5 {
+	font-size: .75rem;
+	font-weight: normal;
+	height: 2.5em;
+}
+
+#lottery-results-pdf-row ol {
+	list-style: none;
+	counter-reset: lottery;
+	margin: 0;
+	padding: 0;
+}
+
+#lottery-results-pdf-row ol > li {
+	counter-increment: lottery;
+	white-space: nowrap;
+}
+
+#lottery-results-pdf-row ol > li::before {
+	content: counter(lottery) ".";
+	color: #999;
+	font-weight: bold;
+	font-size: .65em;
+	text-align: right;
+	width: 2rem;
+	margin-right: 0.2rem;
+	display: inline-block;
+}
+
+kbd {
+	font-family: SFMono-Regular, Consolas, "Liberation Mono", Menlo, Courier, monospace;
+	font-size: .75rem;
+	font-weight: bold;
+	font-style: normal;
+	padding: 3px 5px;
+	line-height: 1;
+	vertical-align: middle;
+	color: #444d56;
+	background-color: #fafbfc;
+	border: solid 1px #c6cbd1;
+	border-bottom-color: #959da5;
+	border-radius: 3px;
+	box-shadow: inset 0 -1px 0 #959da5;
+	display: inline-block;
+}
+
+kbd + kbd {
+	margin-left: .15rem;
+}
+
+@media print {
+	@page {
+		margin: .5in .25in;
+	}
+
+	:root {
+		font-size: 12px;
+	}
+
+	#lottery-results-header.toolbar {
+		display: none;
+	}
+
+	#lottery-results-pdf-generator-container > #lottery-results-pdf-generator-main {
+			/* remove top padding when printing since there's already a page margin */
+		padding: 0 2rem 5rem 2rem;
+	}
+
+	#lottery-results-pdf-row {
+		gap: 0;
+	}
+
+	#lottery-results-pdf-row > div {
+		padding-right: 0;
+	}
+}
```

Comment:
suggestion (non-blocking): Not to belabor the point, but please stick with a __consistent__ style of indentation. I have no opinion regarding tabs vs spaces.

Reply:
So valid